### PR TITLE
Merge changes from 1.2.x → Master

### PR DESCRIPTION
**Pull request by wxtim**
**Check List**

- [x] THis is a merge change.

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -8,6 +8,12 @@ ones in. -->
 
 ## __cylc-rose-1.3.0 (<span actions:bind='release-date'>Awaiting Release</span>)__
 
+### Fixes
+
+[#229](https://github.com/cylc/cylc-rose/pull/229) -
+Fix bug which stops rose-stem suites using the new `[template variables]` section
+in their `rose-suite.conf` files.
+
```
Comment (oliver-sanders) [inline CHANGES.md:16]: This was added to the wrong section on the 1.2.x branch.

**Review by MetRonnie — APPROVED** [review]

**Comment by wxtim** [comment]
> 🤔
> 
> image

It's "→" - I'm not trying to insert rude words in hidden characters. I wasn't.

**Comment by MetRonnie** [comment]
May want to take that character out of the commit subject/message when merging

**Comment by MetRonnie** [comment]
I'm going to merge this. The changelog is now correct

**Pull request merged**